Auth0/WebAuth.swift

```diff
@@ -426,11 +433,11 @@ public protocol AuthTransaction: AuthCancelable {
      - parameter options: options recieved in the openUrl method of the `AppDelegate`
      - returns: if the url was expected and properly formatted otherwise it will return false.
     */
-    func resume(_ url: URL, options: [A0URLOptionsKey: Any]) -> Bool // In the next major, remove `options`
+    func resume(_ url: URL, options: [A0URLOptionsKey: Any]) -> Bool
 
 }
 
-extension AuthTransaction {
+public extension AuthTransaction {
```

Comment:
Is this required to be public now?

Reply:
The protocol is public. If the extension isn't public, the `resume(_ url:options:) -> Bool` default method implementation that it provides won't be available outside of the package.

Reply:
Makes sense 👍